Easy/03 - Longest Word.js

```diff
@@ -47,3 +47,13 @@ function LongestWord(sen) {
     }
     return longestWord;
 }
+
+
+// Using Array sort() Method
+function LongestWord(sen) { 
+    var words = sen.match(/[a-z0-9]+/gi);
+    var sorted = words.sort(function(a,b){
```

Comment:
Still missing a space before your function brace, but I'll add eslint to the repo so things like this get marked automatically.